enable/examples/demo/enable/canvas_demo.py

```diff
@@ -5,9 +5,9 @@
 
 class Demo(DemoFrame):
 
-    def create_component(self):
+    def _create_component(self):
```

Comment:
part of #500 
All this required to fix was this leading underscore so I figured it was fine to add with this PR.

The issue name is misleading, as the other examples in the comments of the issue are still independently broken.  they need to overwrite the `_create_component` method.  I will update issue and try to fix those in a later PR

Reply:
This makes sense. Thank you!